### ISSUE TITLE
Feature request: add option to read hidden rows

### ISSUE THREAD

**Issue by 0xced** [MarkPflug/Sylvan.Data.Excel#185]
The `ExcelDataReaderOptions` already has the `ReadHiddenWorksheets` option. It would be nice to also have a `ReadHiddenRows` option. It should default to `true` in order not to break compatibility with existing code.

**Comment by MarkPflug** [comment]
Would probably make sense to also allow skipping hidden columns.

**Comment by 0xced** [comment]
Absolutely! Should probably be `ReadHiddenCells` instead of `ReadHiddenRows`. Would you be interested in a pull request?

**Comment by MarkPflug** [comment]
No PR needed. I think I have it working for .xlsx, just need to add the other formats and add some test coverage.

**Comment by MarkPflug** [comment]
Finally implemented in 0.4.28. Added `ReadHiddenRows`, but not `ReadHiddenColumns`, which can be easily achieved by using the `DbDataReader.Select` extension method provided in `Sylvan.Data`
